Demo/src/main/java/com/braintreepayments/demo/PayPalRequestFactory.java

```diff
@@ -118,17 +123,124 @@ public static PayPalVaultRequest createPayPalVaultRequest(
         return request;
     }
 
+    private static Float calculateItemTotal(
```

Comment:
Is this necessary? It looks like we are hardcoding all of these values, so we should be able to calculate these ourselves?